### PR TITLE
Auto tempo

### PR DESCRIPTION
**Pull request by claytonhenrylewis**
Tempo detection

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,24 +12,83 @@ <h1 id="logo">
...
         <div id="tempo-input" class="hidden">
             <h3>Input Tempo: </h3>
-            <!-- TODO: add input options -->
+            <form>
+                <div class="input-container">
+                    <input type="number" name="tempo" min="1" max="999" value="4">
+                </div>
+                <input type="submit">
+            </form>
+            <div id="detect-tempo-div" style="display: block">
```
Comment (calebflynn13) [inline index.html]: Can you add the style properties to the main.css?

Comment (calebflynn13) [reply index.html]: should look like
` #detect-tempo-div {
     display: block;
}`

Comment (claytonhenrylewis) [reply index.html]: sure thing

**Review by calebflynn13 — APPROVED** [review]
Looks good. The only thing I would consider changing is having them try again if there is a large difference in time between each set of beats.

**Pull request merged**